test/test_Wochenende_SLURM.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+## A SLURM sbatch script which is part of the Wochenende Pipeline https://github.com/MHH-RCUG/Wochenende for testing
+## To run it, cd into the test folder this file is in, modify the bastemp directory
+# and use sbatch command to schedule the job on your slurm cluster.
+
+# set partition
+#SBATCH -p normal
+
+# set run on x MB node only
+#SBATCH --mem 40000
+
+# set run x cpus
+#SBATCH --cpus-per-task 4
+
+# set max wallclock time
+#SBATCH --time=47:00:00
+
+# set name of job
+#SBATCH --job-name=Wochenende_test
+
+# Add miniconda3 to PATH
+. /mnt/ngsnfs/tools/miniconda3/etc/profile.d/conda.sh
+
+# Activate env on cluster node
+conda activate wochenende >> /dev/null
+
+pytest --basetemp=/ngsssd1/rcug/tmp/ -n 4
```

Comment:
We probably should add a `srun -c $cpus pytest `...  and define `cpus=4` in the line above to the use the parameter in SLURM. 
`#SBATCH --cpus-per-task 4` is not sufficient weirdly.